### PR TITLE
Made required changes. Tests are passing.

### PR DESCRIPTION
**Pull request by meta-morpho-sys**
Please  review and comment if there is anything else I could do here.

### CODE REVIEW

**Review by i-hardy — APPROVED** [review]
:shipit:

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -3,47 +3,44 @@
...
-  <div class="field">
-    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
-    <%= f.password_field :current_password, autocomplete: "off" %>
-  </div>
 
-  <div class="actions">
+  <div class="actions"></br>
     <%= f.submit "Update" %>
-  </div>
+  </div></br>
```
Comment (paulmillen) [inline app/views/devise/registrations/edit.html.erb]: I believe br's don't need '/' ?

Comment (meta-morpho-sys) [reply app/views/devise/registrations/edit.html.erb]: yes, thank you for your suggestion. I will do that in future.

**Review by aballal — APPROVED** [review]

**Review by petrakh — APPROVED** [review]
:shipit:

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (dbugsy) [reply app/views/devise/registrations/edit.html.erb]: `</br>` shouldn't really be necessary - all the styling and positioning should be done by CSS anyways. If you remove them, does it break?

**Review by dbugsy — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (meta-morpho-sys) [reply app/views/devise/registrations/edit.html.erb]: Thank you Dan for our comment.

When I remove them, the fields become all attached (like compressed). Of course, when the CSS will be done, the problem will cease existing. If you are ok with this I will proceed to making the change.

**Review by dbugsy — APPROVED** [review]

**Pull request merged**